CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.13.4
+
+- enabel options for hostPID, iscsiDirHostPath and iscsiDirHostPathCheckDirectory for [Talos](https://talos.dev) iscsi support.
```

Comment:
Update the comment here to reflect the current options.

As an FYI, at some point I think I’ll remove these altogether. Using chroot and nsenter basically nullify the need for the iscsi mounts altogether but I haven’t tested fully to ensure that.

Reply:
Done. Understandable, no problem at all. I will keep an eye on the project releases to see when will git rid of these variables.